packages/api/models/rules.js

```diff
@@ -257,9 +252,6 @@ class Rule extends Manager {
       newRuleItem.state = 'ENABLED';
     }
 
-    newRuleItem.createdAt = createdAt || Date.now();
-    newRuleItem.updatedAt = Date.now();
```

Comment:
Thanks for this! It actually simplifies one of my tests in another PR because I don't have to overwrite the updatedAt after creation.

Reply:
I actually have to revert this change to this otherwise a bunch of unrelated unit tests break and I don't want to explode the size of this PR. But this change should still allow you to do what you need

```
    newRuleItem.createdAt = item.createdAt || Date.now();
    newRuleItem.updatedAt = item.updatedAt || Date.now();
```